packages/util/src/get-pingback-id.ts

```diff
@@ -4,7 +4,7 @@ let pingbackId = ''
 const idLength = 16
 
 /* istanbul ignore next */
-const noUUIDRandom = () => {
+export const noUUIDRandom = () => {
```

Comment:
out of curiosity, why "no" as a prefix?

Reply:
@gpadin-cp it's not a uuid random id, which requires a somewhat large library, it's just a drop in replacement for a random id